src/esl-iter.c

```diff
@@ -308,13 +308,13 @@ esl_list_iter_next_with_size_correction(esl_list_iter *iter, efi_guid_t *type,
 		return -1;
 	}
 	if (iter->offset < 0) {
-		efi_error("iter->offset (%zd) < 0", iter->offset);
+		efi_error("iter->offset (%jd) < 0", (uintmax_t) iter->offset);
```

Comment:
off_t is a signed type, so `intmax_t` is more clear on these, I think.  (Also, project style is no space after cast.)  This applies to all the iter->offset handling below as well.

Reply:
Updated, thanks.